### PR TITLE
Update set-adfsproperties.md with corrupt/missing EnableOAuthDeviceFlow Parameter details

### PR DESCRIPTION
**Pull request by X-Guardian**
The `Set-AdfsProperties` document page has a corrupted version of the `EnableOAuthDeviceFlow` parameter in its parameter list, and is also missing the parameter details for this.

### CODE REVIEW

**Comment by tseward** [comment]
Looks good.

File: docset/windows/adfs/set-adfsproperties.md [hunk 1]
```diff
@@ -493,6 +493,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### - EnableOAuthDeviceFlow
```
[the remark this answers is not in the document]
Comment (X-Guardian) [reply docset/windows/adfs/set-adfsproperties.md]: Done

**Comment by v-raaba** [comment]
@e0i is it expected that PR is merging to `live`? We are used to merging PRs to `master`.

We publish to the `live` branch from `master` everyday at 10:30 PST & 3:30 PST. It would be preferrable if contributers PRed against `master`. This would ensure that `master` is in Sync with `live`.

**Comment by e0i** [comment]
@X-Guardian Please change the target branch of this PR to `master` so that we can proceed with the PR review. Thanks.

**Comment by X-Guardian** [comment]
How do I do that without starting the pull request again?

All I've done is selected the `Edit` button on the `Set-AdfsProperties` [Documentation Page](https://docs.microsoft.com/en-us/powershell/module/adfs/set-adfsproperties) which is configured to edit the md file in the `live` branch. If you want editors to do anything different,that link needs to be changed.

**Comment by e0i** [comment]
@X-Guardian 

Try editing this _pull request_ via the Edit button at the top right corner. You may then click to the branch link and change the target. Thanks.

**Comment by X-Guardian** [comment]
OK done, but someone still needs to get that `Edit` button changed, or all future edits from anyone are all going to end up with PRs linking to the `live` branch.

**Pull request merged**